src/Image.php

```diff
@@ -41,6 +41,10 @@ class Image extends AbstractImage
      */
     private $palette;
 
+    /**
+     * @phpstan-param MetadataBag<mixed> $metadata
+     * @psalm-param MetadataBag $metadata
```

Comment:
Why does Psalm need this? It is exactly the same information as in the method signature.

Reply:
Psalm parses the `@phpstan-param` too and is confused by the generic notation `<mixed>`. I was only able to make both tools succeed by specifing both `@phpstan-` and `@psalm-`. I think if both tags are present every tool only uses its own tag.

Reply:
Hm, I see.